### PR TITLE
[FEATURE] Include actions in tables

### PR DESCRIPTION
**Pull request by sidohaakma**
* Tests for actions, getters and mutations
* Tests for components (methods)

### CODE REVIEW

File: test/unit/specs/store/actions.spec.js [hunk 1]
```diff
@@ -159,11 +222,866 @@ describe('actions', () => {
...
+        'sourceLabel': 'DNBC'
+      }
+
+      td.when(get('/api/v2/LifeCycle_Harmonizations/' + harmonization)).thenResolve(apiResponseMock)
+      td.replace(api, 'get', get)
+
+      const options = {
+        payload: harmonization,
+        expectedMutations: [
+          {type: SET_HARMONIZATION_DATA, payload: apiResponseMock},
```
Comment (marikaris) [inline test/unit/specs/store/actions.spec.js]: Unexpected trailing comma

File: test/unit/specs/store/actions.spec.js [hunk 2]
```diff
@@ -159,11 +222,866 @@ describe('actions', () => {
...
```
Comment (marikaris) [inline test/unit/specs/store/actions.spec.js]: Unexpected trailing comma

File: test/unit/specs/store/actions.spec.js [hunk 3]
```diff
@@ -159,11 +222,866 @@ describe('actions', () => {
...
+        ],
+        state: mockedState
+      }
+
+      utils.testAction(actions.__GET_HARMONIZATIONS__, options, done)
+    })
+
```
Comment (marikaris) [inline test/unit/specs/store/actions.spec.js]: Block must not be padded by blank lines, I think you removed the }) by accident

**Review by marikaris — CHANGES_REQUESTED** [review]
I'll open a pull request to your branch

**Comment by marikaris** [comment]
Clicking on folder results in console error

File: src/store/actions.js [hunk 4]
```diff
@@ -2,22 +2,83 @@ import api from '@molgenis/molgenis-api-client'
...
       commit(SET_TREE_DATA, data)
     }, error => {
       commit(SET_ERROR, error)
     })
+  },
+  [GET_CORE_VARIABLES] ({state, commit, getters}, treeId) {
+    const treeLeaf = getters.getRawTreeData.find((item) => {
+      return item.key === treeId
+    })
+    const variables = Object.keys(treeLeaf.variables).map(function (variable) {
```
Comment (mark-de-haan) [inline src/store/actions.js]: Logic like this is a typical getter, you use rawTreeData to retrieve a leaf, then get the variables from that leaf.
Seeing as you duplicate this code in a later action as well, I would suggest writing a getter or a util function.

Makes the actions easier to test as well ;)

Comment (sidohaakma) [reply src/store/actions.js]: Will be refactored in getters-test PR

**Review by marikaris — APPROVED** [review]

**Pull request merged**